modules/ROOT/partials/misc/browser-compatibility.adoc

```diff
@@ -11,15 +11,30 @@
 
 {productname} may not function as expected on unlisted browsers. Try out the {productname} examples on {companyurl} to determine any functional limitations of an unlisted browser.
 
+=== Supported web browsers table
+
 [cols=",^,^,^",options="header"]
 |===
-|Browser |Windows |Mac |GNU/Linux
-|Chrome |{tick} |{tick} |{tick}
-|Firefox |{tick} |{tick} |{tick}
-|Edge |{tick} |- |-
-|Safari |- |{tick} |-
+|Browser                       |Windows |macOS   |GNU/Linux
+|Chrome (current version)      |{tick}  |{tick}  |{tick}
+|Chrome (penultimate version)  |60 days |60 days |-
+|Chrome ESR (current version)  |{tick}  |{tick}  |{tick}
```

Comment:
Technically chrome is not "ESR", that's the term Firefox use. Google call theirs "extended stable":
https://support.google.com/chrome/a/answer/9027636?hl=en

Reply:
Indeed. To quote my just-now-committed commit message:

>Chrome’s longer-term stable release is called ‘Extended stable’ not ‘Extended Support Release’. The table and the following notes have been corrected accordingly.